logstash-core/lib/logstash/api/commands/node.rb

```diff
@@ -77,6 +80,24 @@ def jvm
         def hot_threads(options={})
           HotThreadsReport.new(self, options)
         end
+
+        private
+        ##
+        # Returns a vertex, decorated with additional metadata if available.
+        # Does not mutate the passed `vertex` object.
+        # @api private
+        # @param vertex [Hash{String=>Object}]
+        # @return [Hash{String=>Object}]
+        def decorate_vertex(vertex)
```

Comment:
the description of the method and its name doesn't seem to be well aligned with what it does.
I suggest either renaming this to something like `decorate_with_cluster_uuids` or change the method to merge all the plugin metadata instead of just the cluster_uuids. what do you think?

Reply:
That name makes sense to me. I will push a new commit with the name change. :)